system-test/system_test_redis_integ.sh

```
@@ -0,0 +1,1230 @@
+if [ -f /.dockerenv ] || [ -n "${DOCKER_CONTAINER:-}" ]; then
+    DOCKER_MODE=true
+    SKIP_REDIS=true  # Redis is started by Docker startup script
+fi
+
+# Parse command line arguments
+while [[ $# -gt 0 ]]; do
+    case $1 in
+        --quick)
+            QUICK_MODE=true
+            TEST_DURATION=10
+            REQUESTS_PER_SECOND=2
```

Shellcheck (0.11.0)

[warning] REQUESTS_PER_SECOND appears unused. Verify use (or export if used externally).

(SC2034)